UnityProject/Assets/Scripts/Doors/Modules/CrowbarModule.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using Doors;
+using UnityEngine;
+
+public class CrowbarModule : DoorModuleBase
+{
+	[SerializeField]
+	private float pryTime = 4.5f;
+
+	public override ModuleSignal OpenInteraction(HandApply interaction)
+	{
+		return ModuleSignal.Continue;
+	}
+
+    public override ModuleSignal ClosedInteraction(HandApply interaction)
+    {
+	    //If the door is powered, only allow things that are made to pry doors. If it isn't powered, we let crowbars work.
+	    if ((master.PowerCheck() && Validations.HasItemTrait(interaction.HandObject, CommonTraits.Instance.CanPryDoor ))
+	        || (!master.PowerCheck() && Validations.HasItemTrait(interaction.HandObject, CommonTraits.Instance.Crowbar)))
```

Comment:
add bolt state to the check

Reply:
Bolt state shouldn't be added to the check. Bolt state should simply return false in DoorStateCanChange in its own module.
The individual modules shouldn't need to reference each other.

Reply:
absolutely right, marked as resolved